go.mod

```diff
@@ -12,7 +12,7 @@ require (
 	github.com/golang-jwt/jwt/v4 v4.5.0
 	github.com/google/uuid v1.3.0
 	github.com/lestrrat-go/jwx v0.9.0
-	github.com/optimizely/go-sdk v1.8.3
+	github.com/optimizely/go-sdk v1.8.4-0.20230216074708-27b2772ccf33
```

Comment:
This should be done using `go mod tidy` and `go mod clean` to automatically update `go.sum` and remove irrelevant indirect dependencies which were needed by go-sdk `1.8.3`.